mycroft/audio/speech.py

```diff
@@ -44,12 +43,10 @@ def handle_speak(event):
 
     Parse sentences and invoke text to speech service.
     """
-    config = Configuration.get()
-    Configuration.set_config_update_handlers(bus)
-    global _last_stop_signal
+    global _last_stop_signal, tts
 
     # if the message is targeted and audio is not the target don't
-    # don't synthezise speech
+    # don't synthesise speech
```

Comment:
In America it's 'synthesize'. Two nations divided by a common language 🤣

Reply:
pycharm autocorrected it!